server/api.py

```diff
@@ -950,7 +950,19 @@ def update_standings(
     except Tournament.DoesNotExist:
         return 400, {"message": "Tournament does not exist"}
 
-    # FIXME Check if all teams in standings are there in rostered teams
+    # incoming team IDs might not match the rostered teams, adding validation to catch this
+    rostered_team_ids = (
+        UCRegistration.objects.filter(event=tournament.event)
+        .values_list("team", flat=True)
+        .distinct()
+    )
+    incoming_standings_team_ids = tournament_details.seeding.values()
+    for incoming_team_id in incoming_standings_team_ids:
```

Comment:
What about the case where some IDs are missing from the rostered IDs? Can we do this using set comparison instead of iterating over a list? I think error handling the whole input at once is nicer than getting the user to fix one thing at a time.

Reply:
Yeah its better to show all incorrect teams at once. 
The rostered IDs come from ultimate central right ? I assumed that the UCRegistrations table would be the source of truth for all teams rostered for an event. Can you please explain the case you're talking about ?

Reply:
Essentially, in the seedings we should not have missing IDs. All IDs present in rostered_team_ids should also be present in the seedings. This kind of `if` loop only checks for extra/wrong team IDs.

Reply:
Thanks for clarifying punch ! I got confused when you said IDs could be be missing from rostered IDs. Yeah I'll validate the incoming team IDs for missing IDs 👌

Reply:
@punchagan I've done something like this, how descriptive do you think the error messages should be ? 
```python
    # [5, 6, 7, 8, 9, 10, 14, 15, 87, 88, 99, 1286] for Sectionals Bangalore
    rostered_team_ids = set(
        UCRegistration.objects.filter(event=tournament.event)
        .values_list("team", flat=True)
        .distinct()
    )
    # Incoming seeding could have the same team twice or a team that's not part of the roster
    # say [5, 5, 6, 7, 8, 10, 14, 15, 10000, 88, 99, 1286]
    incoming_standings_team_ids = set(tournament_details.seeding.values())
    missing_teams = rostered_team_ids -  incoming_standings_team_ids
    
    if missing_teams:
        return 400, {
            "message": f"Rostered teams for {tournament.event.title} are {rostered_team_ids}\n{missing_teams} are missing from the seeding !"
        }
```

Also, when creating a new tournament using `TournamentManager`, i think it'd be helpful to show the rostered teams right ?

Reply:
This is the view when creating tournaments currently, the same team could get added twice, team not part of roster could get added. I think its better to help the user on the tournament manager page by showing the rostered teams. Thoughts @punchagan @Joe2k 
![image](https://github.com/india-ultimate/hub/assets/39093537/099eec81-d61c-48c9-a227-48a1207b6a0a)

Reply:
Yep so ideally we would want a drag and drop system to rearrage the seedings in the frontend. But i tried doing that with solidnd package but its super complex and time consuming to take care of all cases. But thats where we wanna reach. 

Basically the idea was we can see all rostered teams in standings hence didnt create seperate view. So now if we make sure we cant add/delete teams while changing the standings then we dont need additional view as we know that in the standings all are the rostered teams and no way to manipulate it

Reply:
So in the implementaion you shared if you can share the missing team_ids / addition team_ids from the backend to the frontend... We can show in the frontend with the id - name using the teamMap and the user can update accordingly